extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -231,6 +231,10 @@ export class ConsentUI {
       // No prompt UI specified, nothing to do
       return;
     }
+    if (this.isPostPrompt_ && !this.parent_.contains(this.ui_)) {
+      toggle(this.ui_, true);
```

Comment:
Do we have to worry about the case that the UI is an amp-element?

https://github.com/ampproject/amphtml/blob/dee22f4002875be9d9cb6e50562cc38b959a2637/extensions/amp-consent/0.1/consent-ui.js#L292-L302

Reply:
in theory yes. But I didn't include the check on purpose because in reality I don't see any use case of using AMP element as the postPromptUI.

Reply:
Ok. I'll leave it to your discretion on whether to add the check.